src/adapters/User/index.test.ts

```diff
@@ -0,0 +1,21 @@
+import { getAuth } from 'adapters/BaseAuth';
+
+import { getUserInfo } from '.';
+
+jest.mock('adapters/BaseAuth');
+
+describe('UserAdapter', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
```

Comment:
do we need this `afterEach` block for this test?

Reply:
@tyrro not sure if I understand correctly, but each time we do mock `jest.mock('adapters/BaseAuth');` at line 5, we should clear mock each test, is that correct? 😅

Reply:
I think we don't need it. There are some tips for using `clear/reset/restore` mock:
https://stackoverflow.com/a/66361187

Reply:
Thanks a lot, @hoangmirs. I removed it in [0bce7dc](https://github.com/manh-t/react-survey/pull/59/commits/0bce7dc60c61d4ca8af568ab14cde75b36f826ab) 🙏